### ISSUE TITLE
Missing creation of client tx cert in deployment guide

### ISSUE THREAD

**Issue by greerben0** [ovrclk/docs#130]
The command to create your client key seems to be missing from the deployment guide. 
`akash tx cert create client --from $KEY_NAME --chain-id $AKASH_CHAIN_ID `

Without it the `akash tx deployment create` command fails with "Error: no certificate file found for account"

This is using
AKASH_VERSION=v0.10.0
AKASH_CHAIN_ID=akashnet-2-testnet-2

**Comment by greerben0** [comment]
Suggest adding this at docs/guides/deploy/README.md#147

## Create the Client Cert

To create the client cert, run:
```sh
akash tx cert create client --from $KEY_NAME --chain-id $AKASH_CHAIN_ID
```

You should see a response similar to:
```json
{
    "height": "156367",
    "txhash": "71033852B9DD46DD33131EE7D1BF24E91AF387A6D31521661EBB536C6969A07F",
    "codespace": "",
    "code": 0,
    "data": "0A190A17636572742D6372656174652D6365727469666963617465",
    "raw_log": "[{\"events\":[{\"type\":\"message\",\"attributes\":[{\"key\":\"action\",\"value\":\"cert-create-certificate\"},{\"key\":\"sender\",\"value\":\"akash1ccyv0x655rqmjgc5hh8a2ds5cr7ta50zcvz9a5\"}]},{\"type\":\"transfer\",\"attributes\":[{\"key\":\"recipient\",\"value\":\"akash17xpfvakm2amg962yls6f84z3kell8c5lazw8j8\"},{\"key\":\"sender\",\"value\":\"akash1ccyv0x655rqmjgc5hh8a2ds5cr7ta50zcvz9a5\"},{\"key\":\"amount\",\"value\":\"5000uakt\"}]}]}]",
    "logs": [
        {
            "msg_index": 0,
            "log": "",
            "events": [
                {
                    "type": "message",
                    "attributes": [
                        {
                            "key": "action",
                            "value": "cert-create-certificate"
                        },
                        {
                            "key": "sender",
                            "value": "akash1ccyv0x655rqmjgc5hh8a2ds5cr7ta50zcvz9a5"
                        }
                    ]
                },
                {
                    "type": "transfer",
                    "attributes": [
                        {
                            "key": "recipient",
                            "value": "akash17xpfvakm2amg962yls6f84z3kell8c5lazw8j8"
                        },
                        {
                            "key": "sender",
                            "value": "akash1ccyv0x655rqmjgc5hh8a2ds5cr7ta50zcvz9a5"
                        },
                        {
                            "key": "amount",
                            "value": "5000uakt"
                        }
                    ]
                }
            ]
        }
    ],
    "info": "",
    "gas_wanted": "200000",
    "gas_used": "93003",
    "tx": null,
    "timestamp": ""
}
```

**Comment by rsum1** [comment]
fixed in #132
